src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -681,14 +682,47 @@ public AlignedAssemblyOrExcuse apply( final Tuple2<Integer, List<SVFastqUtils.Fa
 
         final List<SVInterval> intervals = new ArrayList<>(allEvidence.size());
         while ( evidenceIterator.hasNext() ) {
-            intervals.add(evidenceIterator.next().getLocation());
+            //intervals.add(evidenceIterator.next().getLocation());
+            intervals.addAll( splitInterval( evidenceIterator.next().getLocation(), params ) );
         }
 
         evidenceRDD.unpersist();
 
         return intervals;
     }
 
+    /** Split SVInterval into length-numIntervals List<SVInterval> with requested overlap **/
+    public static List<SVInterval> splitInterval( final SVInterval interval, final Params params ) {
+        List<SVInterval> splitIntervals;
+        final int originalLen = interval.getLength();
+        if( originalLen <= params.maxIntervalLength ) {
+            splitIntervals = new java.util.ArrayList<SVInterval>();
+            splitIntervals.add(interval);
+        }
+        else {
+            // Split interval into numIntervals SVIntervals. Nominally each has length exactly equal to maxLen, and
+            // overlap at least equal to requested amount. Matching condition, up to rounding error:
+            //     originalLen ~ maxLen * numIntervals - overlapLen * (numIntervals - 1)
+            final int numIntervals = (int)Math.ceil( (originalLen - params.splitIntervalMinOverlap)
+                    / (double)(params.maxIntervalLength - params.splitIntervalMinOverlap ) );
+            // If the interval is *slightly* larger than params.maxIntervalLength, you may end up with an absurd amount
+            // of overlap. In this case, shorten the split intervals a bit
+            final int maxOverlap = Math.max( params.maxIntervalLength / 3, params.splitIntervalMinOverlap );
+            final int overlap = (numIntervals * params.maxIntervalLength - originalLen) / (numIntervals - 1);
+            final int splitIntervalLen = (numIntervals == 2 && overlap > maxOverlap) ?
+                    (originalLen + maxOverlap) / 2:
+                    params.maxIntervalLength;
+            // Have numIntervals and splitIntervalLen, so make the intervals
+            final double startShift = (originalLen - splitIntervalLen) / (double)(numIntervals - 1);
```

Comment:
That's fair. I guess I wasn't anticipating numIntervals being that large but maybe it could get there in some cases.